### PR TITLE
Add couchdb_peruser

### PR DESCRIPTION
**Pull request by KlausTrainer**
Please also review [couchdb_peruser](https://github.com/apache/couchdb-peruser).

### CODE REVIEW

**Comment by kxepal** [comment]
+1 \o/

P.S. Ignore travis failure - it's because of docs. I should merge https://github.com/apache/couchdb/pull/329 once it'll be green.

File: rebar.config.script [hunk 1]
```diff
@@ -30,6 +30,7 @@ DepDescs = [
     {couch_plugins,     "couch-plugins",    {branch, "master"}},
     {couch_event,       "couch-event",      {branch, "master"}},
     {couch_stats,       "couch-stats",      {branch, "master"}},
+    {couchdb_peruser,   "peruser",          {branch, "master"}},
```
Comment (rnewson) [inline rebar.config.script]: naming convention is couch_ :)

**Comment by rnewson** [comment]
has the new repo been created and populated? does this work for clusters?

File: rebar.config.script (hunk 1 above)
Comment (KlausTrainer) [inline rebar.config.script]: Fixed it. Thanks.

**Comment by KlausTrainer** [comment]
Yes. Please see the repository here: https://git-wip-us.apache.org/repos/asf?p=couchdb-peruser.git.

Regarding clusters: I've made some efforts to get it 2.0-ready. E.g., it now uses fabric for dealing with user databases, and I added integrations tests that go against the clustered HTTP interface.

**Pull request closed without merging**